src/main/scala/scalismo/utils/Random.scala

```diff
@@ -28,7 +28,8 @@ import org.apache.commons.math3.random.MersenneTwister
  *
  * @param seed The seed for the Random number generator
  */
-case class Random(seed: Long) {
+
+private[utils] case class RandomNumberGenerator(seed: Long) {
 
   implicit val breezeRandBasis: RandBasis = new RandBasis(new ThreadLocalRandomGenerator(new MersenneTwister(seed)))
```

Comment:
While you are at it: this should be changed to `RandBasis.withSeed(seed)`, see #212